src/term/mod.rs

```diff
@@ -1774,7 +1774,7 @@ impl ansi::Handler for Term {
 
     #[inline]
     fn save_cursor_position(&mut self) {
-        trace!("CursorSave");
+        trace!("save cursor");
```

Comment:
Following the other log messages, this should be `saving cursor`. I think that `saving cursor position` might be even better.